packages/core/admin/_internal/node/core/plugins.ts

```diff
@@ -0,0 +1,136 @@
+import os from 'node:os';
+import path from 'node:path';
+import { getModule, PackageJson } from './dependencies';
+import { loadFile } from './files';
+import { BuildContext, CreateBuildContextArgs } from '../createBuildContext';
+
+const CORE_PLUGINS = [
+  '@strapi/plugin-content-manager',
+  '@strapi/plugin-content-type-builder',
+  '@strapi/plugin-email',
+  '@strapi/plugin-upload',
+];
+
```

Comment:
are this the core plugins that the admin is dependent on?

Reply:
yeah it's the same list as `INTERNAL_PLUGINS` – ideally it' be nice to share them